packages/jetpack-common-interfaces/src/Renderable_Interface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Renderable Interface
+ *
+ * @package jetpack-common-interfaces
+ */
+
+namespace Jetpack\Assets;
```

Comment:
Let's change the namespace here - Assets doesn't make much sense. `Utils`? `Common`, `Base`, `Interfaces` instead?